dataland-qa-service/src/main/kotlin/DatalandQaService.kt

```diff
@@ -1,15 +1,20 @@
 package org.dataland.datalandqaservice
 
+import org.dataland.datalandbackendutils.email.EmailSender
 import org.dataland.datalandqaservice.configurations.OpenAPIConfiguration
 import org.springframework.boot.autoconfigure.SpringBootApplication
 import org.springframework.boot.runApplication
 import org.springframework.context.annotation.ComponentScan
+import org.springframework.context.annotation.FilterType
 
 /**
  * Class to define the spring boot application
  */
 @SpringBootApplication
-@ComponentScan(basePackages = ["org.dataland"])
+@ComponentScan(
+    basePackages = ["org.dataland"],
+    excludeFilters = [ComponentScan.Filter(type = FilterType.ASSIGNABLE_TYPE, classes = [EmailSender::class])],
```

Comment:
Why is thie excluded?

Reply:
Because creation of the EmailSender cause a problem if no envs for mailjet login are set. Which is the case for most services.

Reply:
Hm ok, and this problems arises, because emailSender is now part of the backendutils which is used everywhere?

Reply:
Exactly

Reply:
Do we still need this exclusion now?

Reply:
Nope. But I thought I already removed it.

Reply:
Ok i will check it

Reply:
Seems to be gone, so i close it